sdk/src/client/node_api/core/mod.rs

```diff
@@ -28,93 +62,93 @@ impl Client {
         Ok(OutputWithMetadata::new(output, metadata))
     }
 
-    /// Request outputs by their output ID in parallel
+    /// Requests outputs by their output ID in parallel.
     pub async fn get_outputs(&self, output_ids: &[OutputId]) -> Result<Vec<Output>> {
```

Comment:
I wonder if we really need 2 functions, instead of one with a bool param. Maybe it's better to have 2 explicit idk.

Reply:
I think it's better to have two

Reply:
Fair enough